### PR TITLE
feat: welcome action added

### LINKED ISSUE #39: feature: add welcome bot
### Describe the solution you'd like
Adding a `welcome bot` is to provide the required information to first-time contributors to Swiftwave. You need to integrate a welcome bot, which will
- On opening the first issue, send a comment welcoming to SwiftWave community and a link to join slack (https://join.slack.com/t/swiftwave-team/shared_invite/zt-21n86aslx-aAvBi3hv1GigVA_XoXiu4Q)
- On opening the first PR, congratulate him for his first contribution, and say to join slack and take a review from the other contributors.

### Additional context
You can check this repo to integrate the welcome bot from wow-actions -> https://github.com/wow-actions/welcome .
We are open to suggestions if you would like to use another bot for welcoming purposes.

### PR DESCRIPTION
**Pull request by AdiDecodes**
Fixes #39
Added Welcome Action in workflows.

### CODE REVIEW

File: .github/workflows/welcome.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  types: [opened]
+jobs:
+ run:
+  runs-on: ubuntu-latest
+  steps:
+   - uses: wow-actions/welcome@v1
+     with:
+      FIRST_ISSUE: |
+       👋 @{{ author }}
+       Welcome to SwiftWave Community! Star ⭐ the main repository [swiftwave](https://github.com/swiftwave-org/swiftwave) to grow this project and join our Slack community [invite link](https://join.slack.com/t/swiftwave-team/shared_invite/zt-21n86aslx-aAvBi3hv1GigVA_XoXiu4Q) to stay updated. Be sure to follow the issue template!
```
Comment (tanmoysrt) [inline .github/workflows/welcome.yml]: We can remove the line `Be sure to follow the issue template!` for now !

File: .github/workflows/welcome.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Welcome
+on:
+ pull_request:
+  types: [opened, closed]
```
Comment (tanmoysrt) [inline .github/workflows/welcome.yml]: I think we need only an `opened` event for pull_request.

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: .github/workflows/welcome.yml (hunk 1 above)
Comment (AdiDecodes) [reply .github/workflows/welcome.yml]: Okay, Doing necessary changes right away!

**Review by tanmoysrt — APPROVED** [review]
Looks fine !

**Comment by tanmoysrt** [comment]
Great work @AdiDecodes :rocket:

**Comment by AdiDecodes** [comment]
> Great work @AdiDecodes :rocket: 

Thanks @tanmoysrt for assigning it to me ✌️

**Pull request merged**